TestProjects/UniversalGraphicsTest_Lighting/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -119,9 +107,6 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/Scenes/152_Lighting_EnlightenEnvironment.unity
     guid: fb8f8e7cc18a5314f8d66d2ddfbfbaec
-  - enabled: 0
-    path: Assets/Scenes/153_Lighting_EnlightenTerrain.unity
-    guid: 677cb6424a7fe3947974395b6c3d70a4
```

Comment:
Same as above. There's a bug to re-enable this scene in our grabbag.

Reply:
https://fogbugz.unity3d.com/f/cases/1330624/

Reply:
thank you